pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -226,3 +226,7 @@ type TriggerList struct {
 func (t *Trigger) GetStatus() *duckv1.Status {
 	return &t.Status.Status
 }
+
+func (t *Trigger) GetCrossNamespaceRef() duckv1.KReference {
+	return *t.Spec.BrokerRef
```

Comment:
could we check if the pointer is `nil` first before dereferencing it? Otherwise we may cause the webhook to panic here